### PR TITLE
Releaseprep wmde.17

### PR DESCRIPTION
**Pull request by rti**

### CODE REVIEW

File: test/specs/repo/extensions/universal-language-selector.ts [hunk 1]
```diff
@@ -5,9 +5,23 @@ describe( 'UniversalLanguageSelector', function () {
...
+		await $( '.page-Main_Page' ).click();
+		// focus search bar again, lang selector should be there now
+		await $( '#searchform input' ).click();
+
+		await $$( '.imeselector' )
+			.filter( async ( selector ) => selector.isClickable() )[ 0 ]
+			.click();
+
+		// we need to use getHTML, getText() is the empty string for some reason,
+		// does it think the element is hidden?
```
Comment (RickiJay-WMDE) [inline test/specs/repo/extensions/universal-language-selector.ts:22]: https://webdriver.io/docs/api/element/getText/
If an element isn't *interactable* it returns empty, which seems frankly ridiculous to me

File: test/helpers/pages/queryservice-ui/queryservice-ui.page.ts [hunk 2]
```diff
@@ -14,7 +14,6 @@ class QueryServiceUIPage extends SubmittablePage {
 	 *
 	 * @param {string} query
 	 * @param {string[]} prefixes - Optional
-	 * @return {void}
```
Comment (RickiJay-WMDE) [inline test/helpers/pages/queryservice-ui/queryservice-ui.page.ts:17]: This introduced a lint warning - please fix

**Review by RickiJay-WMDE — CHANGES_REQUESTED** [review]

File: test/specs/repo/extensions/universal-language-selector.ts (hunk 1 above)
Comment (rti) [reply test/specs/repo/extensions/universal-language-selector.ts]: Thanks for clarification, added a comment about that.

File: test/helpers/pages/queryservice-ui/queryservice-ui.page.ts (hunk 2 above)
Comment (rti) [reply test/helpers/pages/queryservice-ui/queryservice-ui.page.ts]: Weird, I think it was removed by some linter run. I definitely did not remove it manually. Thanks for the catch. Readded.

**Review by RickiJay-WMDE — APPROVED** [review]

**Pull request merged**